trusted-committer-article.md

```diff
@@ -0,0 +1,232 @@
+# InnerSource Learning Path: Trusted Committer
+
+## Introduction
+
+- tbd
+
+## Introducing The Trusted Committer Role
+
+The Trusted Committer (TC) role is one of the key roles in an InnerSource
+community. It is both a challenging and a rewarding role to fulfill and it is
+instrumental for the success of any InnerSource community. 
+
+Generally speaking, the TC role is defined by its responsibilities, rather than
+by its privileges. On a very high level, a TC represents the interests of both
+their InnerSource community and the products the community is building. So as
+a TC, you'll have both tech oriented and community oriented responsibilities
+and we'll explore both of these dimensions in the following sections.
+
+Before we go into the details of what a TC actually does, let's spend some time
+contrasting the TC role to other roles in IS and why we think the name is both
+apt and important.
+
+- Contributor
+    - makes contributions
+    - sometimes part of the community
+    - sometimes send by another part of the organization to develop a feature
+      for it. This is why contributors are sometimes referred to as Guests. 
```

Comment:
Not sure, @rrrutledge . I think _Host_ indeed refers to the whole community (I'd like to stick with _community_ rather than _team_), while the interpretation of _Guest_ depends on the context. In our case at Bosch, where we mostly have _individual_ contributors, _team_ or _community_ don't really apply for the _Guest_. In most other instances I know of, the _Guest_ refers to another team who has an interest to contribute a change.

Reply:
I've heard host/guest used both ways described above. I like Georg's suggestion.

Reply:
We all like being clear about _Host **Team**_ and _Guest **Team**_ terminology when talking about the team.  Nice suggestion!

Reply:
_Host_ and _Guest_ have already been defined in the introductory video (in a way that is compatible with what we're describing here).  No need to redefine, you can just start using the terms.

Reply:
> Maybe the Host/Guest analogy is not that apt, after all. Or maybe we should use _Host Team_, _Host_ (the TC), _Guest_ (individual contributor) and _Guest Team_?!

Be careful using Host to mean TC because you can have Hosts who aren't TCs and TCs who aren't part of the Host team.